idseq_pipeline/version/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.4.0'
```

Comment:
Please add a note for version 1.5.0 to the release notes in the top level README.

Here, add a comment reminding the next person who bumps the version to document it in the readme.